tests/test_export_great_expectations.py

```diff
@@ -0,0 +1,423 @@
+import logging
+import logging
+import os
+import sys
+
+from typer.testing import CliRunner
+
+from datacontract.cli import app
+from datacontract.export.great_expectations_converter import to_great_expectations
+from datacontract.model.data_contract_specification import \
+    DataContractSpecification
+
+logging.basicConfig(level=logging.DEBUG, force=True)
+
+
+def test_cli():
+    runner = CliRunner()
+    result = runner.invoke(app, [
+        "export",
+        "./examples/export/datacontract.yaml",
+        "--format", "jsonschema"
+    ])
+    assert result.exit_code == 0
+
+
+def test_to_great_expectation():
+    data_contract_file = "./examples/export/datacontract.yaml"
+    file_content = read_file(data_contract_file=data_contract_file)
```

Comment:
I think it might be cleaner to define fixtures which return the example data contracts, that way il would be easier to reuse them in potential other tests you'll define in the future.